mlflow/store/artifact/cli.py

```diff
@@ -119,5 +124,29 @@ def download_artifacts(run_id, artifact_path, artifact_uri):
     artifact_location = artifact_repo.download_artifacts(artifact_path)
     print(artifact_location)
 
+
+@commands.command("archive-hdfs-artifacts")
+@click.option("--run-id", "-r", help="Run ID to archive the artifacts")
+def archive_hdfs_artifacts(run_id):
+    """
+    Pack into an hadoop archive a folder on HDFS. Only HdfsArtifactStore supported
+    """
+    if run_id is None:
```

Comment:
Here you can use 'required=True' in click.option to make a parameter mandatory.

Reply:
Done

Reply:
Since the parameter is now required in the click.option, we can remove the if I think.

Reply:
Done